nix/sov-celestia-cw.nix

```diff
@@ -16,11 +52,11 @@ let
             cp -r ${../modules} $out/modules
             cp -r ${../mocks} $out/mocks
             cp -r ${../proto} $out/proto
+
             cp -r ${sovereign-sdk-src} $out/vendor/sovereign-sdk
-            cp ${../Cargo.lock} $out/Cargo.lock
 
-            cat ${../Cargo.toml} > $out/Cargo.toml
-            cat ${../.cargo/config.toml} >> $out/Cargo.toml
+            cp ${cargo-lock-file} $out/Cargo.lock
+            cat ${cargo-toml-file} > $out/Cargo.toml
```

Comment:
Is this to replace the exisiting `$out/Cargo.toml`?

Reply:
In the new code, we would do string manipulation in Nix to replace the git URLs with file paths in Cargo.toml. We then write the output to the intermediary source code, which has the Cargo.toml replaced with the modified version.

Reply:
Sounds good. is there any reason for using `cat` and `cp` together?

Reply:
Ah, that was an artifact from the previous edit. Nice catch, fixed now.